static/app/views/settings/project/projectKeys/details/loaderSettings.tsx

```diff
@@ -40,12 +40,21 @@ export function LoaderSettings({keyId, orgSlug, project, data, updateData}: Prop
   const values = requestPending
     ? optimisticState
     : {
-        browserSdkVersion: data.browserSdkVersion,
+        browserSdkVersion:
+          // "latest" was an option that we do not users to select anymore. It will be phased out when version v8 of
+          // the SDK is released, meaning we want to map the backends response to v7 when it responds with "latest".
```

Comment:
nit/feel free to ignore:

```suggestion
          // the SDK is released, meaning we want to map the backend's response to v7 when it responds with "latest".
```